Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc

```diff
@@ -24,9 +24,30 @@ using namespace edm;
 
 TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
   : fromDDD_( p.getParameter<bool>( "fromDDD" )),
-    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
-    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
```

Comment:
See here:
https://cmssdt.cern.ch/SDT/cgi-bin//showAddOnLogs.py/slc6_amd64_gcc481/www/tue/7.4-tue-02/CMSSW_7_4_X_2015-01-06-0200/addOnTests/

Though I wionder why jenkins/github does not flag this as RED on 
https://cmssdt.cern.ch/SDT/html/showIB.html for today's 2Am IB build.